packages/react/src/themes/teams/components/Dropdown/dropdownItemStyles.ts

```diff
@@ -17,13 +20,51 @@ const dropdownItemStyles: ComponentSlotStylesInput<DropdownItemProps, DropdownVa
       ...getBorderFocusStyles({
         siteVariables,
         isFromKeyboard: p.isFromKeyboard,
+        borderRadius: 0,
       })[':focus'],
       ...(!p.isFromKeyboard && {
         color: v.listItemColorHover,
         backgroundColor: v.listItemBackgroundColorHover,
+        ...(p.header && {
+          [`& .${DropdownItem.slotClassNames.header}`]: {
+            color: v.listItemColorHover,
+          },
+        }),
+        ...(p.content && {
+          [`& .${DropdownItem.slotClassNames.content}`]: {
+            color: v.listItemColorHover,
+          },
+        }),
       }),
     }),
   }),
+  image: ({ props: p }): ICSSInJSStyle => ({
+    margin: `${pxToRem(3)} ${pxToRem(12)} ${pxToRem(3)} ${pxToRem(4)}`,
+    // using !important because:
+    // a ' > :not(:last-child)' selector affecting the margin-right style
+    marginRight: `${pxToRem(12)} !important`,
+  }),
+  header: ({ props: p, variables: v }): ICSSInJSStyle => ({
+    fontSize: v.listItemHeaderFontSize,
+    // if the item doesn't have content - i.e. it is header only - then it should use the content color
+    // or Should the default behavior of handling a generic list be changed, so that the "content" property
+    // is populated by default and not the header?
```

Comment:
Let's change `DropdownItem.create()` to follow suit with `ListItem.create()`.  The string should be passed to the `content` prop, instead of the `header` prop.